client/src/App.js

```diff
@@ -1,13 +1,43 @@
 import "./App.css";
+import React, { useState } from "react";
+
+import UploadComponent from "./components/UploadComponent";
+import Search from "./components/Search";
+import Container from "./components/Container";
+import Footer from "./components/Footer";
+
+const App = () => {
+  const [data, setData] = useState([]);
```

Comment:
"data" is a very generic name for a piece of state - can you think of a name that better describes _what_ data is stored here?